eslint.config.mjs

```diff
@@ -22,10 +22,6 @@ const config = createConfig([
       // design system react
       'packages/design-system-react/src/components/Icon/icons/*.tsx',
       'packages/design-system-react/src/components/Icon/icons/index.ts',
```

Comment:
We will continue to ignore these auto-generated files so this PR closes https://github.com/MetaMask/metamask-design-system/issues/657